modules/puzzles/util.py

```diff
@@ -77,3 +78,31 @@ def remove_duplicate_locations(locations):
             results.append(l)
             storedLocations.append(normalizedL)
     return results
+
+
+def brute_force_coordinates(digits_list):
+    # Add token to seperate latitude and longitude later
+    digits_list.append(',')
+    perms = [''.join(x) for x in list(map(list, permutations(digits_list, len(digits_list))))]
```

Comment:
Volgens mij is dit overbodig. Kun je niet hier onder door de permutations heen lopen?

Reply:
permutations(digits_list) returned een list met daarin tuples zoals (['52', '6'], ['3'], ['78'], [','], ['947']) in deze regel wordt dat omgeschreven tot '526378,947'. Heeft het een voordeel om dat in de loop eronder te doen?

Reply:
Ja; het is sneller en gebruikt minder geheugen. `permutations` returned een iterable object, wat on the fly de waardes uitrekent. Hier forceer je dat naar een lijst waar je dan vervolgens strings van maakt om er dan nog een lijst van te maken. Het maken van strings, en het vervolgens splitten op een `,` is relatief traag. Daarnaast verlies je ook informatie over waar het ene element begint en de ander eindigt op deze manier